### PR TITLE
Adding initial settings for black and pytest tests

### LINKED ISSUE #76: Adding pytest and pytest-black dynamic and static settings
adding initial settings for using pytest and avoid conflicts with other PRs of the common basis of PRs related to tests using pytest

### PR DESCRIPTION
**Pull request by maayanhd**
- Added the pytest.ini file to support pytest tests
- modified Pipfile and Pipfile.lock as part of the pytest and black
  installation

  Setting the ground for pytest and black tests in the following PRs.

closes #76 

***in case the pipfile is doing troubles I suggest running this command:***
 - **pipenv lock --pre --clear**
(after running - vagrant up -> vagrant ssh -> cd /vagrant)

After it will be merged we can run this command for example: 
***pipenv run pytest --black***

### CODE REVIEW

**Review by EdDev — CHANGES_REQUESTED** [review]
We have now entered a new stage of the development, where each change needs to be proven to work before taken in (or even reviewed by maintainers).
Therefore, could you please:
- Split this PR into two, one for adding black (great initiative!) and another for pytest.
- Make sure both are triggered by the CI (in the black case, it will require changes of the code).
- For pytest, it is enough to run it on an empty test pool, it should just not detect any and we should be able to see it on the CI logs.

**Comment by guy9050** [comment]
any update on this? @maayanhd @Shoval-Ovadia

**Comment by Shoval-Ovadia** [comment]
Hi @guy9050,
I opened a new PR for adding 'Pytest':
https://github.com/beyond-io/picATrip/pull/82

**Comment by maayanhd** [comment]
For now, it's on hold unless @nadavspinco will take the 'black' part, and as @Shoval-Ovadia said, she's handling the 'pytest' part, as we agreed in our prior meeting, and updated in the slack either. The main reason for that is better dividing work and prioritizing other tasks over this PR although it's a small PR.

**Comment by maayanhd** [comment]
closing due to splitting work and opening PR #92  and merging PR #82

**Pull request closed without merging**
